lblod/ember-rdfa-editor-lblod-plugins
# Ensure compatibility with `@appuniversum/ember-appuniversum` 2.16

### Overview
The recent 2.16 release of the `@appuniversum/ember-appuniversum` package contains some changes to its component structure (i.e. move to the `.gjs` format). Helper components such as `AuList::Item` have been removed from their own files and moved into their parents component files (in this case `AuList`). Check out https://github.com/appuniversum/ember-appuniversum/releases/tag/v2.16.0 and https://github.com/lblod/ember-rdfa-editor/pull/993 for more information.

This PR replaces the direct invocations of `AuList::Item` by either using the yield `Item` component from the `AuList` component, or by an `li` tag with class `au-c-list__item`.

### How to test/reproduce
- Start the dummy application
- Check if the UI is unchanged (especially in regards to the sidebar)



### Checks PR readiness
